### DISCUSSION TITLE
Trouble setting up the camera

### DISCUSSION THREAD

**Discussion by John-Zephyr** [ramokz/phantom-camera#352] [Question / Help Wanted] [answered]
I downloaded the phantom camera and followed the instructions given(camera3d, phantomhost etc.) However, when I try to play the scene to see if it works, I get an error message which reads: Attempt to call function 'pcam_host_added' in base 'null instance' on a null instance. While I know the basics of programming and have followed a few godot tutorials, I'm kind of lost on what to do here. I realize that I probably just forgot to do some basic set up, or something like that, but I would really appreciate some guidence, since I can't really find a youtube video on that part.

**Comment by ramokz** [comment]
The error implies that it cannot find the `PCamHost` node, which doesn't make a lot of sense, given that's what is calling the function in the first place. Hard to tell what it might be without any additional context.

Could you either share a screenshot / screen recording of your scene structure, or the Godot project with as few assets and files as possible, just to remove potential variables?

**Reply by John-Zephyr** [reply, marked as answer]
Thank you very much for the swift reply! I closed my laptop and went for a walk shortly after posting my question. I just opened up godot again so that I could send you a screenshot, but now it's working normally for some reason. I don't know why or how that is, but since the problem resolved itself, I'll call that a happy ending. Still, it's great to know that you're so quick and eager to respond and help me. I'll be sure to return if I ever have more troubles. In the meantime, I wish you the best.

**Reply by ramokz** [reply]
No problem! Hope the addon will help you make cool things :slightly_smiling_face:
